### PR TITLE
Layout/spacing

### PR DESCRIPTION
**Pull request by mhewson**

### CODE REVIEW

File: docs/layout/spacing/README.md [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+-  `sm` - applies the spacing **for the small breakpoint (768px <= X < 992px)** 
+-  `md` - applies the spacing **for the medium breakpoint (992px <= X < 1232px)**
+-  `lg `- applies the spacing **for the large breakpoint ( > 1232px)**
+
+For example, `.cdr-space-inset-eighth-x@md` will apply if the user’s screen is currently at the medium breakpoint (within 992px and 1232px wide).
+
+For more information on breakpoints and responsive design, see the [Responsive Layout](../responsive) article.
+
+To add right margin to the Cedar button component at the medium breakpoint and up, but bottom margin to at the small breakpoint and below:
+
```
Comment (RatulaREI) [inline docs/layout/spacing/README.md]: Where is the table?

File: docs/layout/spacing/README.md [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+
+For a full list of space utility classes, see the [CSS Utility Class](../../components/utilities/) document.
+
```
Comment (RatulaREI) [inline docs/layout/spacing/README.md]: This is the last sentence of the article.

File: docs/layout/spacing/README.md [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+<cdr-doc-example-code-pair :background-toggle="false" :sandbox-data="$page.frontmatter.sandboxData" >
+
+```vue
+  <div>
+    <cdr-button 
+     full-width="@xs"
```
Comment (RatulaREI) [inline docs/layout/spacing/README.md]: This table goes above the last sentence.

File: docs/layout/spacing/README.md [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+    A sample Cedar button
+    </cdr-button>
+    <cdr-button 
+    modifier="secondary"
+    full-width="@xs"
+    > 
+    Another sample Cedar button
+    </cdr-button>
+  </div>
+
```
Comment (RatulaREI) [inline docs/layout/spacing/README.md]: I don't see the sample Cedar button screenshot in the doc.

File: docs/layout/spacing/README.md (hunk 2 above)
Comment (RatulaREI) [reply docs/layout/spacing/README.md]: But, I think we can delete this sentence from here, as we are already adding it at the end.

File: docs/layout/spacing/README.md (hunk 4 above)
Comment (mhewson) [reply docs/layout/spacing/README.md]: there is a code block at the bottom --

```
<cdr-button 
  space="
    cdr-mb-space-one-x@xs
    cdr-mb-space-one-x@sm
    cdr-mr-space-one-x@md
    cdr-mr-space-one-x@lg
  "
> 
 Your buttons text
</cdr-button>
``` 

I replaced this with a sandbox that actually demos what is going on

**Pull request merged**